content/chat/rooms/index.textile

```diff
@@ -0,0 +1,115 @@
+---
+title: Rooms
+meta_description: "Use rooms to organize your chat traffic."
+product: chat
+languages:
+  - javascript
+---
+
+Rooms are used to organize your message traffic into logically separate 'rooms'. A room can represent a 1:1 chat between an agent and a customer, a direct message between two users in a chat room, a large group conversation, or the chat section of a livestream with thousands of users.
+
+You also control which features are enabled in a room when you create one. For instance, you might only want to utilize typing indicators in direct messages between users.
+
+h2(#create). Create or retrieve a room
+
+Users send messages to a room and subscribe to the room in order to receive messages. Other features, such as indicating which users are online, or which users are typing are all implemented against a room object.
+
+A @room@ is created, or an existing one is retrieved from the @rooms@ collection using the "@rooms.get()@":https://sdk.ably.com/builds/ably-labs/ably-chat-js/main/typedoc/interfaces/Rooms.html#get method:
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream');
+```
+
+When you create or retrieve a room using @rooms.get()@, you need to choose which additional chat features you want enabled in that room. This is configured by passing a "@RoomOptions@":/_LINK object as the second argument. In addition to setting which features are enabled, @RoomOptions@ also configures the properties of the associated features, such as the timeout period for typing indicators.
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream', {PresenceOptions, ReactionsOptions});
+```
+
+You can also use a property for each @RoomOption@ to configure whether the default settings should be used:
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream', {TypingOptions: DefaultTypingOptions});
+```
+
+Enable each feature using its associated option:
+
+| Feature | @RoomOption@ | Default settings |
+| "Presence":/chat/rooms/presence | @PresenceOptions@ | @DefaultPresenceOptions@ |
+| "Occupancy":/chat/rooms/occupancy | @OccupancyOptions@ | None |
+| "Typing indicators":/chat/rooms/typing | @TypingOptions@ | @DefaultTypingOptions@ |
+| "Room reactions":/chat/rooms/reactions | @ReactionsOptions@ | None |
+
+h3(#release). Release a room
+
+Releasing a room allows the underlying resource to be collected. Once "@rooms.release()@":/_LINK has been called, the room will be unusable and a new instance will need be created using "@rooms.get()@":#create if you want to reuse it.
+
+Releasing a room may be optional for many applications. If you have multiple transient rooms, such as in the case of a 1:1 support chat, then it is may be more beneficial.
```

Comment:
This is just my subjective opinion, I'd highly recommend devs to always release when finished with a room, unless in special cases where they only use one room and they're not in a single-page app setting.